opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -2272,6 +2294,12 @@ namespace Opm
             connPI += np;
         }
 
+        //  Sum with communication in case of distributed well.
+        const auto& comm = this->parallel_well_info_.communication();
+        if (comm.size() > 1)
+        {
+            comm.sum(wellPI, np);
+        }
```

Comment:
Is there a reason you did not add this step to the [MultisegmentWell](https://github.com/OPM/opm-simulators/blob/aa38a2264d319e735aade14742f9e4897c13abe8/opm/simulators/wells/MultisegmentWell_impl.hpp#L1238-L1244) case?

Reply:
Multisegment well is not treated, yet. Apart from the places where it was needed to satisfy necessary interface changes.

Reply:
But they sure look very similar.